jo-rdan/prop-lite
# bug(ui): fix signin page

#### What does this PR do?
- redesign the signin form
- adds a button to submit details and link to the index.html

#### Description of Task to be completed
- A user should be able to signin using a form and on clicking the submit button, they are redirected to home page.

#### How should this be manually tested?
- start the server
- use the url `http://127.0.0.1:5500/UI/pages/signin.html?` to open the signup page
- enter details in fields and click signin
- you should be successfully redirected to home.html

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
[#167152693](https://www.pivotaltracker.com/story/show/167152693)

#### Screenshots (if appropriate)
[image](https://user-images.githubusercontent.com/51836855/60848449-83844f80-a19b-11e9-826a-edccb7418fd1.png)
